tests/acceptance/features/coreApiShareOperationsToShares2/shareAccessByID.feature

```diff
@@ -91,19 +90,6 @@ Feature: share access by ID
       | 2               | 0          | 404              |
 
 
-  Scenario Outline: accept a share using empty share Id
```

Comment:
These could be valid test scenarios. Just remove step `Given parameter "shareapi_auto_accept_share" of app "core" has been set to "yes"`. IMO, the tests are in expected failure list not because of that Given step.